app/javascript/components/App.js

```diff
@@ -178,10 +177,13 @@ class App extends Component {
               let userInfo = this.state.personalInfo.find(user => user.id === parseInt(localid))
               let contactInfo = this.state.emergencyContacts.filter(contact => contact.user_id === parseInt(localid))
               return (
-                <>
+                <div id="profile-contents">
```

Comment:
When styling in CSS, typically people don't target `id`. Not saying you need to change it, just an FYI. You should be able to use `className` in the future.